internal/component/faro/receiver/server.go

```diff
@@ -80,7 +80,6 @@ func (s *server) Run(ctx context.Context) error {
 	})
 
 	mw := middleware.Instrument{
-		RouteMatcher:     r,
```

Comment:
Not sure about the impact of this change (This is related to the update of dskit : https://github.com/grafana/dskit/commit/27d7d41066d39e4ee64a0ae8a964518bac1b9f1e)